modules/nodes-nodes-grace-periods-graceful-shutdown.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: PROCEDURE
+[id="nodes-nodes-grace-periods-graceful-shutdown_{context}"]
+= Configuring grace periods
+
+You can create and applying a custom `KubeletConfig` to configure grace periods and associated pods.
+Set the `shutdownGracePeriod` and `shutdownGracePeriodCriticalPods` timings  for normal and critical pods.
+Label one or more `MachineConfigPools` to apply the custom kubelet configuration to one or more nodes in a pool.
+
+By default, `shutdownGracePeriod` and `shutdownGracePeriodCriticalPods` are set to 0.
+
+In this example, kubelet will delay node shutdown for 30 minutes. 20 minutes (30-10) are reserved for normal pods, and 10 minutes for pods for "critical" pods (pods with priority >=2000000000). 
```

Comment:
and 10 minutes for pods for "critical" pods 
should corrected:  and 10 minutes are reserved for "critical" pods 
or: and 10 minutes for pods of "critical"

Reply:
Clarified.